packages/jupyter-ai/jupyter_ai/handlers.py

```diff
@@ -114,13 +114,21 @@ def get_chat_user(self) -> ChatUser:
             "jupyter_collaboration", False
         )
 
+        login = getpass.getuser()
+        initials = login[0].capitalize()
+
         if collaborative:
-            return ChatUser(**asdict(self.current_user))
+            chat_user_kwargs = {
+                # set in case IdentityProvider doesn't return initials, e.g.
+                # JupyterHub (#302)
+                "initials": initials,
```

Comment:
Should `initials` in this case be derived from `self.current_user` instead of `getpass.getuser()`?

Reply:
Great suggestion! Just implemented this.